src/viam/components/component_base.py

```diff
@@ -43,4 +43,10 @@ async def do_command(self, command: Mapping[str, ValueTypes], *, timeout: Option
         raise NotImplementedError()
 
     async def get_geometries(self) -> List[Geometry]:
+        """
+        Get all geometries associated with the Component, in their current configuration, in the frame of the Component.
+
+        Returns:
+            List[Geometry]: The geometries associated with the Component.
+        """
         raise NotImplementedError()
```

Comment:
can we raise the viam unimplemented error here instead? that way we'd get the actual grpc unimplemented status code